### PR TITLE
feat: add Ory Network CORS documentation

### PR DESCRIPTION
**Pull request by zepatrik**
This is very minimal right now, but I was not sure what else to add.

### CODE REVIEW

**Review by kmherrmann — DISMISSED** [review]
Nice, thanks!
Once we ship the corresponding Console screen, let's amend the docs accordingly.

File: docs/guides/cors.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ory patch project <your-project-id> \
+  --replace '/cors_public/enabled=true' \
+  --replace '/cors_public/allowed_origins=["https://*.foobar.ory"]'
+```
+
+You can find more information about the `ory patch` command in the [Ory CLI documentation](./cli/10_ory_cli.mdx).
+
+## CORS on custom domains
+
+Custom domains can be configured to use specific CORS settings. Head over to the [Custom Domains](./custom-domains.mdx#cors)
```
Comment (vinckr) [inline docs/guides/cors.mdx:42]:
```suggestion
You can configure custom domains to use specific CORS settings. Head over to the [Custom Domains](./custom-domains.mdx#cors)
```

Avoid passive when possible

**Review by vinckr — CHANGES_REQUESTED** [review]
Avoid the We form. [Avoid we](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we)from MS styleguide

+ a few more minor issues

File: docs/guides/cors.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+CORS can be enabled using the Ory CLI:
+
+```shell
+ory patch project <your-project-id> \
+  --replace '/cors_public/enabled=true' \
+  --replace '/cors_public/allowed_origins=["https://*.foobar.ory"]'
+```
+
+You can find more information about the `ory patch` command in the [Ory CLI documentation](./cli/10_ory_cli.mdx).
```
Comment (vinckr) [inline docs/guides/cors.mdx:38]:
```suggestion
You can find more information about the `ory patch` command in the [Ory CLI documentation](./cli/ory-patch).
```

What did you want to link here? there 10_ory_cli doesnt exist I think

**Comment by aeneasr** [comment]
What do you think of moving the CORS document under the API section? https://www.ory.sh/docs/api/

**Comment by zepatrik** [comment]
> What do you think of moving the CORS document under the API section? https://www.ory.sh/docs/api/

I don't really care, wherever you think it makes most sense.

**Comment by aeneasr** [comment]
ok let's move it there i think it makes sense

**Pull request merged**